src/DTO/Creditor.php

```diff
@@ -26,4 +27,14 @@ public function getName(): ?string
     {
         return $this->name;
     }
+
+    public function setOrgId(string $orgId): void
```

Comment:
`orgId` must be renamed into `organisationId`, in the private member, setter and getter.